docker/testing/ubuntu18_j11.docker

```diff
@@ -80,6 +80,11 @@ RUN echo 'export ANT_HOME=/usr/share/ant' >> /home/cassandra/.bashrc && \
     echo 'export JAVA11_HOME=/usr/lib/jvm/java-11-openjdk-amd64' >> /home/cassandra/.bashrc && \
     echo 'export JAVA_HOME=$JAVA8_HOME' >> /home/cassandra/.bashrc
 
+ENV ANT_HOME=/usr/share/ant \
```

Comment:
you can't really build this docker image anymore, so not sure we actually need these changes here